java/src/main/c++/jni_spark_vw.cc

```diff
@@ -1,13 +1,17 @@
 #include "jni_spark_vw.h"
-#include "vw_exception.h"
+
+#include <algorithm>
+#include <exception>
+
 #include "best_constant.h"
-#include "util.h"
 #include "config/cli_options_serializer.h"
+#include "config/options.h"

```

Comment:
nit: group includes in different directories separately from local ones?

Reply:
Its still technically a local search path include though. Clang-format also treats them all as the same unit too due to being a local include so I think this would be hard to change